test/test-remote/testutils/logs.ts

```diff
@@ -66,10 +66,16 @@ export async function queryLoki(baseUrl: string, queryParams: URLSearchParams) {
   const headers = enrichHeadersWithAuthToken(url, {});
 
   const options = {
-    retry: 0,
+    // Allow up to two retries in the event of spurious timeouts or similar errors.
+    retry: 2,
     throwHttpErrors: false,
     searchParams: queryParams,
-    timeout: httpTimeoutSettings,
+    // Use a 5s request timeout rather than the configured 60s default.
+    // We want the poll loop to retry quickly if there is a timeout.
+    timeout: {
+      connect: httpTimeoutSettings.connect, // inherit default
+      request: 5000,
```

Comment:
I want to say 5 seconds are super tight.

For some kinds of queries this might simply be too low. Say, response generation for a specific query is expected to take 10 seconds then we will need to change this again. When the time comes :).

I'm good with the change for now.